src/decomp.jl

```diff
@@ -273,6 +273,7 @@ function factorize_eigen(A::ITensor,
     Ris = uniqueinds(A,IndexSet(Linds...))
     A2 = A*prime(dag(A),Ris)
     if delta_A2 != nothing
+      println("Applying eigen perturbation")
```

Comment:
Looks like this needs to be removed. Otherwise looks good to me.

Reply:
Whoops! Good catch